docs/google/calibration.md

```diff
@@ -0,0 +1,200 @@
+then query for the current or previous state of the calibration.
+Calibrations are also available for past jobs.
+
+A calibration object is a dictionary from metric name (see below) to the value
+of the metric.  Note that the value of the metric is also usually a dictionary
+(for instance, from qubit or qubit pair to a float value).
+
+## Retrieving calibration metrics
+
+Calibration metrics can be retrieved using an engine instance or with a job.
+
+```
+import cirq.google as cg
+
+# Create an Engine object to use.
+# Replace YOUR_PROJECT_ID with the id from your cloud project.
+engine = cg.Engine(project_id=YOUR_PROJECT_ID, proto_version=cg.ProtoVersion.V2)
+processor = engine.get_processor(processor_id=PROCESSOR_ID)
+
+# Get the latest calibration metrics.
+latest_calibration = processor.get_current_calibration()
+
+# If you know the timestamp of a previous calibration, you can retrieve the
+# calibration using the timestamp in epoch seconds.
+previous_calibration = processor.get_calibration(CALIBRATION_SECONDS)
+
+# If you would like to find a calibration from a time-frame, use this.
+calibration_list = processor.list_calibration(START_SECONDS, END_SECONDS)
+
+# If you know the job-id, you can retrieve the calibration that the job used.
+job = engine.get_job("projects/" + PROJECT_ID
+                   + "/programs/"+PROGRAM_ID
+                   + "/jobs/" + JOB_ID)
+job_calibration = cg.EngineJob(cg.JobConfig(), job, engine).get_calibration()
+
+# The calibration can be iterated through using something like the following.
+for metric_name in latest_calibration:
+  print(metric_name)
+  print('------')
+  for qubit_or_pair in latest_calibration[metric_name]:
+     # Note that although the value is often singular,
+     # the metric_value is of the type list and can have multiple values     
+     metric_value = latest_calibration[metric_name][qubit_or_pair]
+     print(f'{qubit_or_pair} = {metric_value}')
+```
+
+Calibration metrics will also soon be available from the 
+[Google Cloud Platform Console](https://console.cloud.google.com).
+
+## Average, Pauli and Purity Error
+
+Several metrics below define average error, pauli error and purity error.
+The average error is equal to one minus fidelity averaged over all possible
+input states.
+
+Pauli error defines decoherence of a single qubit in one of the Pauli channels
+X, Y, or Z.  If the errors are distributed in a uniform distribution across all
+three axes, the probability of applying an erroneous Pauli gate X, Y, or Z will
+be the Pauli error divided by three.  The Pauli error and average error are
+related by a multiplicative factor based on the number of qubits.
+
+See Table 1 on page 11 of the
+[https://arxiv.org/abs/1910.11333](Supplementry Information)
+document for a description and comparison between average error, Pauli error,
+and depolarization error.
+
+The [Purity](https://en.wikipedia.org/wiki/Purity_(quantum_mechanics)) of the
```

Comment:
I would consider referring instead to the "unitarity", which is the decay rate per gate (or cycle) of the purity. We choose to normalize this decay rate to be on the same scale as the "total error" (which is also a rate per gate). You can mention that in casual parlance we often refer to the unitarity as the "purity error".

Reply:
Done.